bundles/CoreBundle/Resources/translations/en.json

```diff
@@ -849,5 +849,9 @@
   "permission_missing": "You need the '%s' permission to perform this action",
   "paste_as_language_variant": "Paste as new language variant",
   "select_language_for_new_document": "Select language for New Document",
-  "source_document_language_missing": "Set Language(Properties) in Source Document"
+  "source_document_language_missing": "Set Language(Properties) in Source Document",
+  "unique_identifier": "Unique identifier",
```

Comment:
The translation keys `unique_identifier` , `invalid_identifier` and `identifier_already_exists` already exist.